controllers/storage.go

```diff
@@ -140,13 +151,13 @@ var ConnectAndQueryObjStore = func(ctx context.Context, log logr.Logger, endpoin
 				"If using an tls S3 connection with  self-signed certs, you may specify a custom CABundle " +
 				"to mount on the DSP API Server via the DSPA cr under the spec.cABundle field. If you have already " +
 				"provided a CABundle, verify the validity of the provided CABundle."
-			log.Error(err, errorMessage)
+			log.Info(errorMessage)
```

Comment:
imho, should be `log.Error`

Reply:
same as this comment https://github.com/opendatahub-io/data-science-pipelines-operator/pull/519#discussion_r1462036825